spec/spec_helper.rb

```diff
@@ -0,0 +1,94 @@
+require 'factory_girl_rails'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.